src/Sulu/Component/Content/Mapper/ContentMapper.php

```diff
@@ -144,7 +149,8 @@ public function __construct(
         SessionManagerInterface $sessionManager,
         EventDispatcherInterface $eventDispatcher,
         ResourceLocatorStrategyPoolInterface $resourceLocatorStrategyPool,
-        NamespaceRegistry $namespaceRegistry
+        NamespaceRegistry $namespaceRegistry,
+        bool $enablePathAttribute = true
```

Comment:
Same question as above, shouldn't we make that an array in case more attributes will be passed in the future?